Workiva/w-mobile-kit
# [MOBILE-2624] Increase flexibility of toast and action sheet

Description
---
Recent changes to Wdesk app revealed need to improve flexibility over action sheet cells, and the WToast component

What Was Changed
---
- You can pass an optional `options` dictionary into `WAction`s to customize the title and subtitle font and color.
   - This allowed us to imitate a disabled cell but still call an action when the user selects it
- A WToastFlexibleView component was created that allows a toast to grow in height to fit the passed in text, while still being capped by a maxHeight property

Testing
---
1. Navigate to `Various Modal View Examples` page in example app
2. Tap `Auto Dismiss Flexible Toast` button, notice that the toast fits the text with consistent vertical padding as the other toasts
3. Tap `Cancel Action Sheet` button, notice that the top cell in action sheet looks disabled but can be tapped
4. Tap `Autosize Sheet` button, notice the purple text in the `Title` cell


---

Please Review: @Workiva/mobile
